test/language/expressions/greater-than-or-equal/bigint.js

```diff
@@ -0,0 +1,13 @@
+// Copyright (C) 2017 Robin Templeton. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: BigInt comparison
+esid: pending
+features: [BigInt]
+---*/
+
+assert(1n >= 0);
+assert(1 >= 0n);
+assert(1n >= 1);
+assert(1 >= 1n);
```

Comment:
Include some test which requires a little bit more calculation, and include a comparison between some non-numeric types and a BigInt.